prow/apis/prowjobs/v1/types.go

```diff
@@ -286,10 +286,39 @@ type ReporterConfig struct {
 }
 
 type SlackReporterConfig struct {
-	Host              string         `json:"host,omitempty"`
-	Channel           string         `json:"channel,omitempty"`
-	JobStatesToReport []ProwJobState `json:"job_states_to_report,omitempty"`
-	ReportTemplate    string         `json:"report_template,omitempty"`
+	Host              *string         `json:"host,omitempty"`
+	Channel           *string         `json:"channel,omitempty"`
+	JobStatesToReport *[]ProwJobState `json:"job_states_to_report,omitempty"`
```

Comment:
Slices are already pointer types, I don't think there is a need to use a double pointer

Reply:
good point, fixed